README.md

```diff
@@ -85,7 +85,7 @@ During the current phase of this project, we only support SPM. We have plans to
 dependencies: [
     .package(
         url: "https://github.com/twilio/twilio-verify-sna-ios.git",
-        .upToNextMajor(from: "0.0.2")
+        .upToNextMajor(from: "0.0.4")
```

Comment:
Why was it bumped to .4 instead of .3?

Reply:
The sample app was 0.0.3 and it could be better to have the same version as the SDK, we can use 0.0.3 or 0.0.4, which one would be better?

Reply:
Changed to 0.0.3